Dockerfile

```diff
@@ -1,23 +1,19 @@
-FROM debian:9
+FROM python:3.5-stretch
 LABEL maintainer="Codimp"
 
-RUN apt-get update && \
-    apt-get install -y \
-    dexdump \
-    python3-dev \
-    python3-pip \
-    libssl-dev \
-    libffi-dev \
-    libxml2-dev \
-    libxslt1-dev \
-    postgresql-client
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y \
+    dexdump=7.0.0* \
+    postgresql-client-9.6=9.6.15* \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
 
 COPY ./requirements.txt /opt/requirements.txt
 RUN pip3 install -r /opt/requirements.txt
 
 RUN useradd -ms /bin/bash exodus
 
-RUN mkdir -p /home/exodus/.config/gplaycli && cp /usr/local/lib/python3.5/dist-packages/root/.config/gplaycli/gplaycli.conf /home/exodus/.config/gplaycli/gplaycli.conf
+RUN mkdir -p /home/exodus/.config/gplaycli && cp /usr/local/lib/python3.5/site-packages/root/.config/gplaycli/gplaycli.conf /home/exodus/.config/gplaycli/gplaycli.conf
```

Comment:
Why this change?

Reply:
Because it wouldn't work without it =)
My guess is that previously we were installing python from debian packages and the python docker image gets a built version; `dist-packages` is specific to [debian packages](https://stackoverflow.com/questions/9387928/whats-the-difference-between-dist-packages-and-site-packages)